custom-configurations/src/main/java/com/sequenceiq/cloudbreak/exception/CustomConfigsException.java

```diff
@@ -0,0 +1,7 @@
+package com.sequenceiq.cloudbreak.exception;
+
+public class CustomConfigsException extends RuntimeException {
```

Comment:
should we rename this? CustomConfigurationsCreationException? this should be more specific definitely

Reply:
Okay, will rename the exception to be more specific